amxmodx/amxmodx.h

```diff
@@ -96,19 +96,17 @@ extern AMX_NATIVE_INFO g_GameConfigNatives[];
 	#endif
 #endif
 
-#if defined(_WIN32)
+#if defined PLATFORM_WINDOWS
 	typedef HINSTANCE DLHANDLE;
 #else
 	typedef void* DLHANDLE;
 	#define INFINITE 0xFFFFFFFF
 #endif
 
-#if defined(_WIN32)
+#if defined PLATFORM_WINDOWS
```

Comment:
As said in the previous review, you can remove those constants, they are redundant with `PLATFORM_` version